interfaces/builtin/serial_port.go

```diff
@@ -208,6 +209,25 @@ func (iface *serialPortInterface) AutoConnect(*snap.PlugInfo, *snap.SlotInfo) bo
 	return true
 }
 
+func (iface *serialPortInterface) HotplugDeviceDetected(di *hotplug.HotplugDeviceInfo, spec *hotplug.Specification) error {
+	if di.Subsystem() == "tty" && strings.HasPrefix(di.DeviceName(), "/dev/ttyUSB") {
```

Comment:
I think this may be more than just `/dev/ttyUSB` interfaces, there's a whole zoo of `tty<somerandomstring>` appearing on various SoC systems.

Reply:
Yes, I'm aware of that, we use "^/dev/tty(mxc|USB|ACM|AMA|XRUSB|S|O)[0-9]+$" pattern for the non-hotplug-aware part of this interface. I just went for a simple ttyUSB" as this is the only one I can reasonably test at the moment.
Anyway, noted, thanks, I'll either extend this with a regex too, or remove completely if we don't want to only partially support hotplug for this interface.